docs/topics/proc_deploying-an-existing-wildfly-swarm-application-to-openshift.adoc

```diff
@@ -0,0 +1,60 @@
+
+[id='deploying-an-existing-wildfly-swarm-application-to-openshift]
+= Deploying an existing {WildFlySwarm} application to OpenShift
+
+You can easily deploy your existing application to OpenShift using the Fabric8 Maven plugin.
+
+.Prerequisites
+
+* A {WildFlySwarm}{ndash}based application
+
+.Procedure
+
+. Add the following profile to the `pom.xml` file in the root directory of your application:
++
+--
+[source,xml,options="nowrap"]
+----
+<profile>
+  <id>openshift</id>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>fabric8-maven-plugin</artifactId>
```

Comment:
Should probably show that specifying `<version>` is necessary.

Reply:
Done.